File: repos/beecrowd/c/bee1010.c
#include <stdio.h>
 
int main() {
    int codigo1, qntd1, codigo2, qntd2;
    double valor1, valor2, total;
    scanf("%d %d %lf", &codigo1, &qntd1, &valor1);
    scanf("%d %d %lf", &codigo2, &qntd2, &valor2);
    total = (qntd1*valor1) + (qntd2*valor2);

    printf("VALOR A PAGAR: R$ %.2lf\n", total);
    return 0;
}

File: repos/beecrowd/python/bee1010.py
codigo_peca1, total_peça1, valor_peça1 = (input()).split()
codigo_peça2, total_peça2, valor_peça2 = (input()).split()
total = (int(total_peça1)*float(valor_peça1)) + \
    (int(total_peça2)*float(valor_peça2))
print(f'VALOR A PAGAR: R$ {total:.2f}')
